support-frontend/assets/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -465,28 +455,17 @@ function PaymentRequestButton(props: PropTypes) {
 					setPrbType('PAY_NOW');
 				}
 
-				const isUkOrEuRecurring =
-					props.contributionType !== 'ONE_OFF' &&
-					(props.countryGroupId === 'GBPCountries' ||
-						props.countryGroupId === 'EURCountries');
-				const shouldShowPrb =
-					Boolean(result?.applePay) ||
-					Boolean(result?.googlePay) ||
-					!isUkOrEuRecurring;
```

Comment:
This logic is now in the new `StripePaymentRequestButton` component that is specific to the contribs checkout page